README.md

```diff
@@ -37,8 +39,37 @@ Then follow installation instructions provided on [torch](https://pytorch.org/ge
 
 ## Usage
 
-### Run training
-...
+Usage pipeline:
+
+- Collect a dataset: set of functions or methods. You can use [`./workflow/dataset_for_tests_java.yml`](./workflow/dataset_for_tests_java.yml) as a template.
+- Create configuration (specifying server and training parameters). You can use [`./workflow/config_for_tests.yml`](./workflow/config_for_tests.yml) as a template.
+  - Below we will discuss how you can fully integrate your own symbolic machine
+- Launch the server manager
+  - `poetry --ansi run poe launch_servers --config path/to/config.yml`
+- Run the training process to get PyTorch model.
+  - `poetry --ansi run poe run_training --config path/to/config.yml`
+- Use ONNX to convert your PyTorch model to ONNX format [(see below how to)](#onnx-conversion).
```

Comment:
This is still not quite right, we do not use ONNX to convert pytorch model to `.onnx`. If you want to be specific, then the conversion happens with `torch.onnx` module of `pytorch` lib:
<img width="371" alt="image" src="https://github.com/user-attachments/assets/0085bf4d-6183-4ca0-bb5e-0731dacd3d79">

However, I advise you to not specify this. Instead we can write something like this:
```
Use `onyx.py`(link) command line tool to convert ...
```

Reply:
done